tools/display/timestamp_diffs.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
```

Comment:
I'd call this module `timestamps`, `offset`, or `triggers`. We can then continue to add similar functionality here.